<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Blend_And_Play</class>
 <widget class="QWidget" name="Blend_And_Play">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2150</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_LFMap">
      <property name="text">
       <string>Left-Front Map</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_LFMap">
      <property name="text">
       <string>/home/arjun/workspace/PanoGen/resources/maps/map_LeftFront_float.xml</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_BrowseLFMap">
      <property name="text">
       <string>Browse..</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_LeftVideo">
      <property name="text">
       <string>Left image</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButton_Play">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_RightVideo">
      <property name="text">
       <string>/home/arjun/workspace/PanoGen/resources/images/right.png</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_BrowseRightVideo">
      <property name="text">
       <string>Browse..</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_RightVideo">
      <property name="text">
       <string>Right image</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_FrontVideo">
      <property name="text">
       <string>/home/arjun/workspace/PanoGen/resources/images/front.png</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_LeftVideo">
      <property name="text">
       <string>/home/arjun/workspace/PanoGen/resources/images/left.png</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_BrowseFPMap">
      <property name="text">
       <string>Browse..</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_FrontVideo">
      <property name="text">
       <string>Front image</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_FRMap">
      <property name="text">
       <string>Front-Right Map</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_FRMap">
      <property name="text">
       <string>/home/arjun/workspace/PanoGen/resources/maps/map_FrontRight_float.xml</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_BrowseFrontVideo">
      <property name="text">
       <string>Browse..</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_BrowseLeftVideo">
      <property name="text">
       <string>Browse..</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_BlendWinWidth">
      <property name="text">
       <string>Blend window width</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_BlendWinWidth">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_Panorama">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>1704</width>
     <height>276</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
